macroslib/src/java_jni/jni-include.rs

```diff
@@ -1361,3 +1361,46 @@ impl<T: SwigForeignClass> SwigFrom<Option<T>> for jobject {
         }
     }
 }
+
+#[swig_from_foreigner_hint = "java.util.Optional<T>"]
```

Comment:
It is not Java way to pass `java.util.Optional<T>` as input argument, see for example:

https://stackoverflow.com/questions/31922866/why-should-java-8s-optional-not-be-used-in-arguments
http://dolszewski.com/java/java-optional-parameters/

The problem is three states instead of two in Rust: analog of `Some`, analog of `None` and `null`.
For output parameters complexity that add null compensated by methods like `Optional.flatMap` and so on things.
But for input parameters it should be just `T`, because of any `T` can be `null`.

Reply:
Fair enough. Made it T.